From 8a23c2485c919e273dd95fc0c4988ee193a9707c Mon Sep 17 00:00:00 2001
From: Sven Joachim <svenjoac@gmx.de>
Date: Sun, 6 Nov 2016 18:31:44 +0100
Subject: [PATCH] debian: Add missing Breaks on old libdpkg-perl to dpkg

Commit 9d7ba99cc3ff84fc553ed39da9d2e4f4008d35b6 renamed the
triplettable file to tupletable and added the necessary dpkg
dependency to libdpkg-perl, but did not forbid the other now broken
combination of old libdpkg-perl and new dpkg.

Closes: #843429
---
 debian/control | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/debian/control b/debian/control
index 66a3ce3..3ad3760 100644
--- a/debian/control
+++ b/debian/control
@@ -53,6 +53,8 @@ Breaks:
  systemtap (<< 2.8-1),
  terminatorx (<< 4.0.1-1),
  xvt (<= 2.1-20.1),
+# Due to renaming of triplettable to tupletable.
+ libdpkg-perl (<< 1.18.11),
 Description: Debian package management system
  This package provides the low-level infrastructure for handling the
  installation and removal of Debian software packages.
-- 
2.10.2

